trunk/SUAVE/Methods/Missions/Segments/Transition/Constant_Acceleration_Constant_Angle_Linear_Climb.py

```diff
@@ -87,8 +87,8 @@ def initialize_conditions(segment):
     
     # Figure out vx
     V  = (vf_mag-v0_mag) 
-    vx = t_nondim *  V  * np.cos(climb_angle) + v0 * np.cos(climb_angle) 
-    vz = t_nondim *  V  * np.sin(climb_angle) + v0 * np.sin(climb_angle)  
+    vx = t_nondim *  V  * np.cos(climb_angle) + v0[0] * np.cos(climb_angle) 
+    vz = t_nondim *  V  * np.sin(climb_angle) - v0[2] * np.sin(climb_angle)  
```

Comment:
The negative sign below is because the z-direction points down. This one can be made positive if the air speed that is input by the user follows the same convention. In theory this should be the case, but like how altitude input is not specified as negative, this can be a bit ambiguous. I have adjusted to follow the aircraft velocity frame and specified that in the assumptions.